web-server/v0.4/unittests

```diff
@@ -6,6 +6,11 @@ if [[ $? -ne 0 ]]; then
   echo "yarn failed!" >&2
   exit 1
 fi
+yarn start &
+if [[ $? -ne 0 ]]; then
```

Comment:
Not sure this makes sense.  I don't think the value of `$?` is set to anything as the `yarn start` is just immediately put in the background because of the trailing `&` on the line.  You might want to check `$!` to see that it got a pid value, and that the pid is of the `yarn start`.

It is your intention to create the `yarn start` process as a background job?

Reply:
Correct, however this is not required now that we have moved e2e testing to the pre-push hook. Beforehand, `yarn start` was started as a background job to allow the e2e tests to run on the local environment while the local dev server runs.

Reply:
I've removed this block in the most recent commit.